extensions/easyvariable/package.json

```
@@ -0,0 +1,193 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "easyvariable",
+  "title": "Easy Variable",
+  "description": "Convert Any Language Text into an English Variable Name",
+  "icon": "extension-icon.png",
+  "author": "chs",
+  "categories": [
+    "Developer Tools",
+    "Data"
+  ],
+  "license": "MIT",
+  "preferences": [
+    {
+      "type": "checkbox",
+      "title": "Switch Translate",
+      "description": "use Google Translate, may need proxy",
+      "required": false,
+      "name": "enableGoogleTranslate",
+      "default": false,
+      "label": "Google Translate"
+    },
+    {
+      "type": "checkbox",
+      "description": "Use Raycast Pro AI for translation",
+      "required": false,
+      "name": "enableRaycastTranslate",
+      "default": false,
+      "label": "Raycast AI"
+    },
+    {
+      "type": "checkbox",
+      "description": "Enable Youdao Web Translate",
+      "required": false,
+      "name": "enableYoudaoTranslate",
+      "default": true,
+      "label": "Youdao Translate"
+    },
+    {
+      "type": "checkbox",
+      "description": "Need zhipuai Api Key,glm-4-flash is free",
+      "required": false,
+      "name": "enableGLMTranslate",
+      "default": false,
+      "label": "GLM Translate"
+    },
+    {
+      "type": "checkbox",
+      "description": "Need OpenAI Api Key,or other OpenAI style Api",
+      "required": false,
+      "name": "enableOpenAITranslate",
+      "default": false,
+      "label": "OpenAI Translate"
+    },
+    {
+      "type": "checkbox",
+      "description": "Need Deepseek Api Key",
+      "required": false,
+      "name": "enableDeepseekTranslate",
+      "default": false,
+      "label": "Deepseek Translate"
+    },
+    {
+      "type": "checkbox",
+      "description": "Enable Tencent Translate SecretId and SecretKey",
+      "required": false,
+      "name": "enableTencentTranslate",
+      "default": false,
+      "label": "Tencent Translate"
+    },
+    {
+      "type": "textfield",
```

Please use the password fields for keys/tokens.

```suggestion
      "type": "password",
```

FYI: https://developers.raycast.com/information/security#data-storage